code/Model/EditableCustomRule.php

```diff
@@ -24,24 +25,24 @@
 class EditableCustomRule extends DataObject
 {
     private static $condition_options = [
-        'IsBlank' => 'Is blank',
-        'IsNotBlank' => 'Is not blank',
-        'HasValue' => 'Equals',
-        'ValueNot' => 'Doesn\'t equal',
-        'ValueLessThan' => 'Less than',
-        'ValueLessThanEqual' => 'Less than or equal',
-        'ValueGreaterThan' => 'Greater than',
+        'IsBlank'               => 'Is blank',
```

Comment:
I don't believe we normally use equal indent for array values?

Reply:
Fixed